src/overlays/actors/ovl_En_Guest/z_en_guest.c

```diff
@@ -20,23 +34,200 @@ const ActorInit En_Guest_InitVars = {
     (ActorFunc)EnGuest_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Guest/EnGuest_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Guest/EnGuest_Destroy.s")
+static ColliderCylinderInit_Set3 colliderInit = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x39, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 10, 60, 0, { 0, 0, 0 } },
+};
+
+static InitChainEntry initChain[] = {
+    ICHAIN_U8(unk_1F, 6, ICHAIN_CONTINUE),
+    ICHAIN_F32(unk_4C, 500, ICHAIN_STOP),
+};
+
+UNK_PTR D_80A50BA4[] = {
+    &D_060005FC,
+    &D_060006FC,
+    &D_060007FC,
+};
```

Comment:
you dont have to make any changes here, just want to let you know to make your life easier that you do not need to make undefined_syms entries for segment addresses when they are in an array. its only needed when referenced directly to get lui/addiu instead of lui/ori

Reply:
I had thought undefined_syms entries were also needed for shiftability, is that not the case?

Reply:
undefined_syms still hardcodes the address. its a work around so that they are treated as symbols for the time being to get the correct code generation, which is not needed for an array of them

Reply:
At least then we have a list of all hardcoded addresses, rather than them being scattered about

Reply:
It's fine but it should be avoided in general, because overall it will be more efficient to only deal with them once the corresponding segments are decompiled and these references are named properly.